tsconfig.json

```diff
@@ -1,30 +1,6 @@
 {
-  "compilerOptions": {
```

Comment:
Then honestly I'd postpone this changes and see another chance for refactoring. This PR obviously makes config simpler, but functionwise everything should be same, and to make that introducing another workaround seems bit off to me. Our bulid pipeline already have sufficient workaround to achieve our goal and I'd like to move way to remove those.